src/main.js

```diff
@@ -98,19 +94,13 @@ function createRandomCover() {
 };
 
 function saveCover() {
-  var savedCover = coverImage.src;
-  var savedTitle = bookTitle.innerText;
-  var savedDescriptor1 = tagline1.innerText;
-  var savedDescriptor2 = tagline2.innerText;
-  var mySavedCover = new Cover (savedCover, savedTitle, savedDescriptor1, savedDescriptor2);
-
-  var isNewCover = true;
-  for(var i = 0 ; i < savedCovers.length ; i++) {
-    if(savedCover === savedCovers[i].cover && savedTitle === savedCovers[i].title && savedDescriptor1 === savedCovers[i].tagline1 && savedDescriptor2 === savedCovers[i].tagline2) {
-        isNewCover = false
-    }
-  }
```

Comment:
Believe this logic needs to remain as well? Otherwise, there is no check for duplicate covers?

Reply:
Same as above, this definitely needs to be in there, must've been the "revert" that I did